tests/test_registration_sidebar.py

```diff
@@ -73,3 +76,143 @@ def test_overview_link(self, driver, session):
         metadata_page.goto()
         metadata_page.side_navbar.overview_link.click()
         assert RegistrationDetailPage(driver, verify=True)
+
+
+@pytest.mark.usefixtures('must_be_logged_in_as_registration_user')
+@markers.dont_run_on_prod
+@markers.core_functionality
+class TestRegistrationOutputs:
+    @pytest.fixture()
+    def registration_guid(self, session):
+        registration_guid = osf_api.get_registration_by_title(
+            'Selenium Registration For Outputs Testing'
+        )
+        return registration_guid
+
+    @pytest.fixture()
+    def registration_details_page(self, driver, registration_guid):
+        registration_details_page = RegistrationDetailPage(
+            driver, guid=registration_guid
+        )
+        registration_details_page.goto()
+        return registration_details_page
+
+    resource_type = ['Data', 'Analytic Code', 'Materials', 'Papers', 'Supplements']
```

Comment:
Could you rename this to `resource_types` since it is a list of things?  And update the name is the `parameterized` calls.  This would also help reduce the confusion between this list variable, and the passed-in `resource_type` arguments to the functions below.

Reply:
Nitpick: could you also move this declaration to just before the marker on `def test_add_new_resource`, where we start to use it?

Reply:
Sure, will make changes

Reply:
Done changes and pushing my changes